appinfo/info.xml

```diff
@@ -8,9 +8,14 @@
 	<dependencies>
 		<owncloud min-version="9.0" max-version="9.1" />
 	</dependencies>
+
 	<description>Calendar GUI for ownCloud's integrated CalDAV server</description>
 	<documentation>
 		<user>http://doc.owncloud.org/server/9.0/user_manual/pim/calendar.html</user>
 	</documentation>
+	<category>productivity</category>
```

Comment:
https://doc.owncloud.org/server/9.0/developer_manual/app/info.html

But you don't seem to be the only one who has never seen them ;-)


Am 2016-06-02 um 18:26 schrieb Georg Ehrke:
>
> In appinfo/info.xml 
> <https://github.com/owncloud/calendar/pull/564#discussion_r65572627>:
>
> >  	<description>Calendar GUI for ownCloud's integrated CalDAV server</description>
> >  	<documentation>
> >  		<user>http://doc.owncloud.org/server/9.0/user_manual/pim/calendar.html</user>
> >  	</documentation>
> > +	<category>productivity</category>
>
> Are this properties documented somewhere?
> I've never seem them in other apps or anywhere in core.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/owncloud/calendar/pull/564/files/3f50ff3217d392ced984e6d5dc00210a056b3d13#r65572627>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ALEnw1QvdXMSgNYFT4U6-XgqGJ2vN012ks5qHwQagaJpZM4Isqi9>.
>

-- 
Jan Klostermann
- The Agile Management Sounding-Board -
Systems Engineer & Mediator
Am Isarkanal 24/15
81379 München

Tel.: +49-89/74563382
Mobil: +49-179/4628819
Sip: +49-89/21551663 <sip:08921551663@sipgate.de>
Skype: jan.klostermann <callto://jan.klostermann/>
Threema: P58VSJUY <threema://add?id=P58VSJUY>
Email: jk@janklostermann.de <mailto:jk@janklostermann.de>
Netzwerke: Xing <https://www.xing.com/profile/Jan_Klostermann>, LinkedIn 
<http://de.linkedin.com/in/janklostermann>